### PR TITLE
Scrapper from quoteload

### LINKED ISSUE #3: Write a scrapper from quoteload

### PR DESCRIPTION
**Pull request by harshraj22**
Added the basic version of web scrapper "scrapper.py" . 
Closes https://github.com/oss2019/pyquotes/issues/3

### CODE REVIEW

File: pyquotes/scrapper.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+from bs4 import BeautifulSoup
+import requests
+import random
+
+#website used :
+parent_link = "https://www.quoteload.com"	
+#for quote of the day :
+day_quote = "https://www.brainyquote.com/quote_of_the_day"
```
Comment (akshatkarani) [inline pyquotes/scrapper.py]: If you are using `brainyquote` then remove I suggest you remove this because the same thing will be there in other scrapper.

**Review by akshatkarani — CHANGES_REQUESTED** [review]
1. It is recommended to have one commit for one task. You should squash your commits. And for further changes I suggest you use `--amend` to make changes in same commit.
2. Tests are failing because you have not followed PEP8 style. Click on `details` to see what changes you should make.

File: pyquotes/scrapper.py (hunk 1 above)
Comment (harshraj22) [reply pyquotes/scrapper.py]: Sure. But quoteland doesn't have any section like quote of the day. So if I scrap from there, the quote won't be random. Is it ok not to have a random quote in quote of the day ?

**Comment by harshraj22** [comment]
> 1. It is recommended to have one commit for one task. You should squash your commits. And for further changes I suggest you use `--amend` to make changes in same commit.
> 2. Tests are failing because you have not followed PEP8 style. Click on `details` to see what changes you should make.

1.I don't use terminal for using github. 
2.Sure, will be done soon.

**Comment by akshatkarani** [comment]
> 1.I don't use terminal for using github.

You should start using it. It will be helpful in the longer run.

**Comment by akshatkarani** [comment]
@harshraj22  Can you please squash all the commits together. Also refer the issue in commit message to which this PR is related.

File: pyquotes/scrapper.py (hunk 1 above)
Comment (akshatkarani) [reply pyquotes/scrapper.py]: Remove the quote of the day section, and add random quote, we will add this feature in the API.

**Comment by harshraj22** [comment]
> @harshraj22 Can you please squash all the commits together. Also refer the issue in commit message to which this PR is related.

How to do so ?

**Comment by akshatkarani** [comment]
> How to do so ?

I am not sure how to do it from the website, I suggest you start using terminal. You can figure out this later, first remove the `quote of the day` section.

File: pyquotes/scrapper.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+def get_quotes(person: (None, str) = None, category: (None, str) = None):
+    # function called without any argument:
+    if isinstance(person, type(None)) and isinstance(category, type(None)):
+        print('too few arguments to get quotes')
+        return None  # can be modified to throw some exception instead
+
+    result = []
+    if not isinstance(person, type(None)):  # person's name is provided
```
Comment (akshatkarani) [inline pyquotes/scrapper.py:16]: It is better to use `person is not None`

Comment (harshraj22) [reply pyquotes/scrapper.py:16]: @akshatkarani  have a look at my commit. Made changes as requested.

Comment (akshatkarani) [reply pyquotes/scrapper.py:16]: There are many instances where you have used `isinstance(x, type(None))`, change all of those.

Comment (harshraj22) [reply pyquotes/scrapper.py:16]: @akshatkarani updated.

**Comment by sunn-e** [comment]
@akshatkarani , good to go?

**Comment by sunn-e** [comment]
I think @harshraj22 , you should send a new PR with requested changes as per @akshatkarani requested. But do not worry, since this is a separate module file, for time being I am approving this. Congrats. Do not forget to claim your swags if you 3 PR are merged, if not, you have time till tomorrow. Hurry.

**Pull request merged**